lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -54,8 +54,6 @@ def initialize(options)
 
     def define_local_variables
       @devices_list = []
-      @pre_start_commands = []
```

Comment:
Why did you remove this? We have some devices such as ivshmem/virtio-fs where we MUST run the daemon on the host before tests and kill it after it.

Reply:
I missed their usage. Commit e82eedb3dfaef22b24a28a4972a0bfe6b4dd8938 restores them.

Reply:
For example: https://github.com/HCK-CI/AutoHCK/blob/master/lib/setupmanagers/qemuhck/devices/ivshmem-doorbell.json